test/style/single_node_test.exs

```diff
@@ -304,4 +304,111 @@ defmodule Styler.Style.SingleNodeTest do
       assert_style("Enum.reverse(foo, bar) ++ bar")
     end
   end
+
+  describe "Enum.any?/1 => Enum.empty?/1" do
```

Comment:
you don't need all these tests -- it's enough to just assert that `! Enum.empty(foo)` rewrites to any?, and vice versa, and same when it's a `not` rather than `!`. the parent / surrounding code is irrelevant to the implementation & a red herring to be describing :) (assignment, if statement, etc)